tests/network/conftest.py

```diff
@@ -312,12 +319,38 @@ def _verify_ipv4():
             else:
                 LOGGER.info("Validated network lane is running against an IPV4 supported cluster")
 
+    def _verify_bgp_env_vars():
```

Comment:
Continuing the previous "lost" thread [1]:

Can you please feedback on:
> The last is due to BGP_CLUSTER_DOMAIN_GROUP , which is specific to BGP tests. However, I do not understand why we need it as an argument. Can you explain?


[1] https://github.com/RedHatQE/openshift-virtualization-tests/pull/1776#discussion_r2330399453

Reply:
This is an identifier of the rack on which the cluster is deployed?

If that is your intention, then I prefer the tests not to know about the rack name. Can we find some alternative?